### ISSUE TITLE
Docs: What events are supported for each type?

### ISSUE THREAD

**Issue by jktravis** [openlayers/openlayers#9771]
Hi, I seem to have a terrible time following what events are used where. Most recently, I was looking at the [ModifyInteraction](https://openlayers.org/en/latest/apidoc/module-ol_interaction_Modify-Modify.html) and trying to bind to the `change` handler using something like:

```javascript
const modifyAction = new Modify({source: this.drawingSource});
modifyAction.on("change", console.log)
```

Inspecting the object, I can see that a listener was set, but changing the feature via the mouse, [as indicated in other modify examples](https://openlayers.org/en/latest/examples/?q=modify), never once called the event. It wasn't until later that I was about to post an SO about how to capture the event, that I found that the correct event is `modifyend`. 

Thinking I just overlooked something in the docs, I tried searching for that term, and I get nothing. I looked on the aforementioned Modify page, and nothing. 

Do I have an incorrect expectation here? Should this be more explicit in the docs? Is there a naming convention that I having found information on? Should _that_ be documented more explicitly? 

Thanks.

**Comment by mike-000** [comment]
It has been fixed in OpenLayers 6 - See #9471 and https://openlayers.org/en/master/apidoc/module-ol_interaction_Modify-Modify.html

**Comment by jktravis** [comment]
Fair enough. Thanks!
